pavelib/quality.py

```diff
@@ -303,20 +320,34 @@ def _get_last_report_line(filename):
     """
     Returns the last line of a given file. Used for getting output from quality output files.
     """
-    with open(filename, 'r') as report_file:
-        lines = report_file.readlines()
-        return lines[len(lines) - 1]
+    file_not_found_message = "Complexity log file not found"
```

Comment:
Since this is a generic method that could get called in different circumstance, a better message might be "File foo not found"